rdrf/rdrf/export/definitions.py

```diff
@@ -0,0 +1,97 @@
+from collections import namedtuple
+
+from registry.genetic import models as genemodels
+from registry.patients import models as patientmodels
+from rdrf import models
+from . import model_exporters, datagroup_exporters, mongo_collection_exporters
+
+
+class GroupDefinition(namedtuple('GroupDefinition', ['name', 'dirname', 'datagroups', 'models', 'collections'])):
+    def __new__(cls, name, dirname, datagroups=tuple(), models=tuple(), collections=tuple()):
+        return super(GroupDefinition, cls).__new__(cls, name, dirname, datagroups, models, collections)
+
+
+ExporterCatalogue = namedtuple('Catalogue', ['datagroups', 'models', 'mongo_collections'])
+ExportDefinition = namedtuple('ExportDefinition', ['exporters_catalogue', 'datagroups'])
+
+
+_MAIN_CATALOGUE = ExporterCatalogue(
+        datagroup_exporters.catalogue,
+        model_exporters.catalogue,
+        mongo_collection_exporters.catalogue)
+
+_CDE_GROUP = GroupDefinition(
+    name='CDEs',
+    dirname='CDE',
+    models = (
+        models.CDEPermittedValue,
+        models.CDEPermittedValueGroup,
+        models.CommonDataElement,
+    ))
+
+_REFDATA_GROUP = GroupDefinition(
+    name='Reference Data',
+    dirname='reference_data',
+    models=(
+        genemodels.Gene,
+        genemodels.Laboratory,
+        genemodels.Technique,
+
+        patientmodels.State,
+        patientmodels.NextOfKinRelationship,
+        patientmodels.AddressType,
+    ))
+
+
+# Export definitons for different type of exports
+
+
+CDE_EXPORT_DEFINITION = ExportDefinition(
+    exporters_catalogue=_MAIN_CATALOGUE,
+    datagroups=(_CDE_GROUP,)
+)
+
+REFDATA_EXPORT_DEFINITION = ExportDefinition(
+    exporters_catalogue=_MAIN_CATALOGUE,
+    datagroups=(_REFDATA_GROUP, _CDE_GROUP)
+)
+
+REGISTRY_EXPORT_DEFINITION = ExportDefinition(
+    exporters_catalogue=ExporterCatalogue(
+        datagroup_exporters.catalogue,
+        model_exporters.registry_catalogue,
+        mongo_collection_exporters.catalogue,),
+    datagroups=(
+        _REFDATA_GROUP,
+        _CDE_GROUP,
+        GroupDefinition(
+            name='Registry Definition',
+            dirname='registry_definition',
+            models = (
+                models.Registry,
+                models.RegistryForm,
+                models.AdjudicationDefinition,
+                models.Section,
+        )),
+        GroupDefinition(
+            name='Registry Data',
+            dirname='registry_data',
+            datagroups=(
+                GroupDefinition(
+                    name='Demographic Data',
+                    dirname='demographic_data',
+                    models = (
+                        patientmodels.Patient,
+                )),
+                GroupDefinition(
+                    name='MongoDB Data',
+                    dirname='mongodb_data',
+                    collections = (
+                        'cdes',
+                        'history',
+                        'progress',
```

Comment:
there's a registry specific fields collection called "registry_specific_patient_data" also in mongo - used by FH only currently